### PR TITLE
Proof read INTER-BANK_COMMUNICATION

### LINKED ISSUE #216: Proof INTER-BANK_COMMUNICATION
Proof read transcribed code of [INTER-BANK_COMMUNICATION](//github.com/chrislgarry/Apollo-11/blob/master/Comanche055/INTER-BANK_COMMUNICATION.agc) against scans

Lines: 184
Page: 1183—1106

Reduced quality scans can be found [here][1]

[1]://ibiblio.org/apollo/ScansForConversion/Comanche055

### PR DESCRIPTION
**Pull request by Feliix42**
This should close #216.

I added a period, reformatted the table at the bottom and reintroduced some typos that were removed during the transcription.

I am not sure about [line 110](https://github.com/chrislgarry/Apollo-11/blob/master/Comanche055/INTER-BANK_COMMUNICATION.agc#L110), though. Should it be changed back to `CAF`? RSB changed it to `CA`, back in 2009.

### CODE REVIEW

File: Comanche055/INTER-BANK_COMMUNICATION.agc [hunk 1]
```diff
@@ -87,7 +87,7 @@ SUPDACAL	TS	MPTEMP
 		ROR	SUPERBNK	# SAVE FBANK IN BITS 15-11, AND
 		XCH	MPTEMP		# SUPERBANK IN BITS 7-5.
 		MASK	LOW10
-		XCH	L		# SAVE REL. ADR. IN BANK, FETCH SUPERBITS
+		XCH	L		# SAVE REL. ADR. IN BANK, FETCH SUPERBITS.
```
Comment (wopian) [inline Comanche055/INTER-BANK_COMMUNICATION.agc]: ✔️

File: Comanche055/INTER-BANK_COMMUNICATION.agc [hunk 2]
```diff
@@ -145,27 +145,27 @@ USPRCADR	TS	LOC		# SAVE A
 # THERE ARE FOUR POSSIBLE SETTINGS FOR CHANNEL 07.  (CHANNEL 07 CONTAINS SUPERBANK SETTING.)
 #
 #					PSEUDO-FIXED	 OCTAL PSEUDO
-# SUPERBANK	SETTING	S-REG. VALUE	BANK NUMBERS	   ADDRESSES
-# ---------	-------	------------	------------ 	   ---------
-# SUPERBANK 3	  OXX	 2000 - 3777	   30 - 37	 70000 - 107777		(WHERE XX CAN BE ANYTHING AND
+# SUPERBANK	SETTING	S-REG. VALUE	BANK NUMBERS	 ADDRESSES
+# ----------	-------	------------	------------ 	  ------------
```
Comment (wopian) [inline Comanche055/INTER-BANK_COMMUNICATION.agc]: ✔️

File: Comanche055/INTER-BANK_COMMUNICATION.agc [hunk 3]
```diff
@@ -145,27 +145,27 @@ USPRCADR	TS	LOC		# SAVE A
...
+# SUPERBANK 3	  0XX	2000 - 3777	   30 - 37	 70000 - 107777		(WHERE XX CAN BE ANYTHING AND
```
Comment (wopian) [inline Comanche055/INTER-BANK_COMMUNICATION.agc]: ✔️

File: Comanche055/INTER-BANK_COMMUNICATION.agc [hunk 4]
```diff
@@ -145,27 +145,27 @@ USPRCADR	TS	LOC		# SAVE A
...
 #										ONLY BANKS 40-43 WILL EVER BE
 #										AND ARE PRESENTLY AVAILABLE)
-# SUPERBANK 5	  101	 2000 - 3777	   50 - 57	130000 - 147777		(PRESENTLY NOT AVAILABLE TO
+# SUPERBANK 5	  101	2000 - 3777	   50 - 57	 130000 - 147777	(PRESENTLY NOT AVAILABLE TO
 #										THE USER)
-# SUPERBANK 6	  110	 2000 - 3777	   60 - 67	150000 - 167777		(PRESENTLY NOT AVAILABLE TO
+# SUPERBANK 6	  110	2000 - 3777	   60 - 67	 150000 - 167777	(PRESENTLY NOT AVAILABLE TO
 #										THE USER)
-# *** THIS ROUTINE MAY BE CALLED BY ANY PROGRAM LOCATED IN BANKS 00 - 27.  I.E., NO PROGRAM LIVING IN ANY
+# *** THIS ROUTINE MAYBE CALLED BY ANY PROGRAM LOCATED IN BANKS 00 - 27.  I.E., NO PROGRAM LIVING IN ANY
```
Comment (wopian) [inline Comanche055/INTER-BANK_COMMUNICATION.agc]: ✔️ `MAYBE`

✨ Can you add a double space between `***` and `THIS`, and again on line below between `SUPERSW.` and `***`

File: Comanche055/INTER-BANK_COMMUNICATION.agc [hunk 5]
```diff
@@ -145,27 +145,27 @@ USPRCADR	TS	LOC		# SAVE A
...
 # SUPERBANK SHOULD USE SUPERSW. ***
 #
-# SUPERSW MAY BE CALLED IN THIS FASHION:
-#	CAF	ABBCON		WHERE -- ABBCON  BBCON  SOMETHING --
+# SUPERSW MAYBE CALLED IN THIS FASHION:
```
Comment (wopian) [inline Comanche055/INTER-BANK_COMMUNICATION.agc]: ✔️

File: Comanche055/INTER-BANK_COMMUNICATION.agc [hunk 6]
```diff
@@ -145,27 +145,27 @@ USPRCADR	TS	LOC		# SAVE A
...
+#	CAF	ABBCON		WHERE -- ABBCON    BBCON   SOMETHIN --
```
Comment (wopian) [inline Comanche055/INTER-BANK_COMMUNICATION.agc]: ✔️ `SOMETHIN`

❌ Only 3 spaces between `ABBCON` and `BBCON`

❌ Only 2 spaces between `BBCON` and `SOMETHIN`

✨ Can you add double spaces after `WHERE`, `--` and `SOMETHIN`

File: Comanche055/INTER-BANK_COMMUNICATION.agc [hunk 7]
```diff
@@ -145,27 +145,27 @@ USPRCADR	TS	LOC		# SAVE A
...
 #	TCR	SUPERSW		(THE SUPERBNK BITS ARE IN THE BBCON)
 #	...	  ...
 #	 .	   .
 #	 .	   .
-# OR IN THIS FASHION:
+# OR IN THIS FASHION :
```
Comment (wopian) [inline Comanche055/INTER-BANK_COMMUNICATION.agc]: ✔️

**Comment by Feliix42** [comment]
Thanks for the feedback! I introduced the changes you requested. :)

**Comment by wopian** [comment]
Thanks! Nice to see so many contributions all of a sudden 😃, as I just don't have the time to do these myself until summer 😢

**Comment by Feliix42** [comment]
No problem, it was kinda fun! Maybe I'll do another if I find the time. :)

**Comment by juanmafx** [comment]
Yes we .



Ante cualquier  duda responda este correo.
Juan Manuel Ramallo
Whatsapp:
+5493513118724



-------- Mensaje original --------
De: Felix Wittwer <notifications@github.com>
Fecha: 1/5/17 3:47 AM (GMT-03:00)
A: chrislgarry/Apollo-11 <Apollo-11@noreply.github.com>
CC: Subscribed <subscribed@noreply.github.com>
Asunto: Re: [chrislgarry/Apollo-11] Proof read INTER-BANK_COMMUNICATION (#291)


No problem, it was kinda fun! Maybe I'll do another if I find the time. :)

**Pull request merged**